barretenberg/cpp/src/barretenberg/vm2/tracegen/lib/lookup_into_rng_chk.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cassert>
+
+#include "barretenberg/vm2/tracegen/lib/lookup_builder.hpp"
+
+namespace bb::avm2::tracegen {
+
+template <typename LookupSettings> class LookupIntoRngChk : public BaseLookupTraceBuilder<LookupSettings> {
```

Comment:
Ah interesting! You were able to use the same class for lookups into u8, u16 etc, because they all use CLK and only change based on the selector. Maybe add a comment?